.golangci.yml

```diff
@@ -40,7 +40,7 @@ issues:
     - path: cloudtest\.go
       linters:
         - golint
-    - path: api/v1alpha2/types\.go
+    - path: api/v1alpha4/types\.go
```

Comment:
this is an exclude rule, shouldn't it exclude v1alpha3?

Reply:
yes, you are right, fixed it